config/config.go

```diff
@@ -151,6 +153,19 @@ func Init(ctx context.Context, config Config, opts InitOptions) (*engine.Engine,
 		return nil, errors.New(errkind.Invalid, "no registries defined", op)
 	}
 
+
+	if config.TimeoutSeconds != 0 && (config.TimeoutSeconds < 5 || config.TimeoutSeconds > 3600) {
+		return nil, errors.New(errkind.Invalid, fmt.Sprintf("Invalid timeoutSeconds value '%d' in .tbrc.yaml. Values must be between 5 and 3600 inclusive", config.TimeoutSeconds), op)
+	}
+
+	var timeout time.Duration
+	if config.TimeoutSeconds != 0 {
+		timeout = time.Duration(config.TimeoutSeconds) * time.Second
+	} else {
+		// default to 60 min timeout when not provided in .tbrc.yml
+		timeout = time.Duration(3600) * time.Second
```

Comment:
Nit, you don't need to explicitly convert numeric constants, i.e. `3600 * time.Second` works